src/openrct2/rct2/S6Importer.cpp

```diff
@@ -1620,6 +1623,28 @@ class S6Importer final : public IParkImporter
         auto withoutFormatCodes = RCT12::RemoveFormatCodes(originalStringView);
         return rct2_to_utf8(withoutFormatCodes, RCT2_LANGUAGE_ID_ENGLISH_UK);
     }
+
+    std::vector<rct_object_entry> GetRequiredObjects()
+    {
+        std::vector<rct_object_entry> result;
+        rct_object_entry nullEntry = {};
+        std::memset(&nullEntry, 0xFF, sizeof(nullEntry));
+
+        int objectIt = 0;
+        for (int16_t objectType = OBJECT_TYPE_RIDE; objectType <= OBJECT_TYPE_WATER; objectType++)
```

Comment:
I take it Scenerio text isn't included for a reason.

Reply:
Yes. Ted and I removed support for that type of object long ago.